app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    @user = User.find_by(username: params[:session][:username])
+    if @user && @user.authenticate(params[:session][:password])
+      session[:user_id] = @user.id
+      redirect_to dashboard_path(id: @user.id)
+    else
+      flash.now[:error] = "Invalid. Try Again."
+      render :new
+    end
+  end
+
+  def destroy
+    session.clear
+    redirect_to "/"
```

Comment:
I'm going to change the path helper in my branch for the landing page, so we don't need to worry about it here